packages/ckeditor5-modern/src/ModernEditor/ModernEditor.tsx

```diff
@@ -59,10 +66,68 @@ const ModernEditor: FC<ModernEditorProps> = ({
 
   const { isSubmitting } = formState;
   const values = getValues();
+
   const images = useMemo(() => files.map(file => URL.createObjectURL(file)), [
     files,
   ]);
 
+  const uploadFiles = async (files: File[]) => {
+    try {
+      setUploadLoading(true);
+      setUploadCurrentLoading(0);
+      setUploadTotalLoading(files.length * 100);
+      let currentTotal: number[] = [];
+
+      // get upload credentials
+      const resultUploadCredentials: Promise<string>[] = files.map(
+        async (file: File) => {
+          return await uploadGetCrendentials({
+            type: file.type.split('/')[0],
+            filename: file.name,
+            apiUrl: uploadConfig?.apiUrl || '',
+            file: file,
+          });
+        }
+      );
+
+      const dataUploadCredentials = await Promise.all(resultUploadCredentials);
+      const url = dataUploadCredentials.map((upload: any) => upload.data.url);
```

Comment:
@nberongoy avoid using `any`, if you know what's the return value please add typings